src/SourceBuild/content/eng/tools/BinaryToolKit/DetectBinaries.cs

```diff
@@ -45,44 +45,24 @@ public static class DetectBinaries
         return unmatchedBinaryFiles;
     }
 
-    private static async Task<List<string>> GetIgnoredPatternsAsync(string targetDirectory)
+    private static async List<string> GetIgnoredPatterns(string targetDirectory)
     {
-        string gitDirectory = Path.Combine(targetDirectory, ".git");
-        bool isGitRepo = Directory.Exists(gitDirectory);
-
-        try 
-        {
-            if (!isGitRepo)
-            {
-                // Configure a fake git repo to use so that we can run git clean -ndx
-                await ExecuteProcessAsync("git", $"-C {targetDirectory} init -q");
-            }
-
-            await ExecuteProcessAsync("git", $"-C {targetDirectory} config --global safe.directory {targetDirectory}");
-
-            string output = await ExecuteProcessAsync("git", $"-C {targetDirectory} clean -ndx");
-
-            List<string> ignoredPaths = output.Split(Environment.NewLine)
-                .Select(line => GitCleanRegex.Match(line))
-                .Where(match => match.Success)
-                .Select(match => match.Groups[3].Value)
-                .ToList();
-
-            if (isGitRepo)
-            {
-                ignoredPaths.Add(".git");
-            }
-
-            return ignoredPaths;
-        }
-        finally
+        return new List<string>
         {
-            // Ensure .git directory is deleted if it wasn't originally a git repo
-            if (!isGitRepo && Directory.Exists(gitDirectory))
-            {
-                Directory.Delete(gitDirectory, true);
-            }
-        }
+            "**/.git/**/*",
+            "**/.dotnet/**/*",
+            "**/.packages/**/*",
+            "**/artifacts/**/*",
+            "**/prereqs/packages/**/*",
+            "src/nuget-client/NuGet.config",
```

Comment:
Why is this included since it's not a binary file?

Reply:
Good point, it was in the VMR's ignore file and I did a copy of that here. I'll remove it.